.golangci.yml

```diff
@@ -1,3 +1,22 @@
+run:
+  tests: true
+  skip-dirs:
+    - bin
+    - vendor
+    - var
+    - tmp
+    - example/*
+
 linters-settings:
   errcheck:
     ignore: fmt:.*,[rR]ead|[wW]rite|[cC]lose,io:Copy
+
+linters:
+  enable-all: true
+  disable:
```

Comment:
a long list of enabled linters is probably better than removing from a changing target imo.